validate.py

```diff
@@ -181,6 +182,8 @@ def check_registration(old, new, op):
             'registration_status': check_registration,
             #auth tokens are never given access
             'auth': say_no,
+            #nonessentials
+            'traveling_from\\.mode': lambda x,y,z: y in ('bus', 'train', 'car', 'plane'),
```

Comment:
Later on, when you add rule for `formatted_address`, be sure to check against changes to `travelliing_from` too: they should not be able to set the whole field to something invalid too.